the-blue-alliance-ios/View Controllers/Events/EventsTableViewController.swift

```diff
@@ -218,7 +218,7 @@ extension EventsTableViewController: TableViewDataSourceDelegate {
         guard let event = dataSource?.object(at: IndexPath(item: 0, section: section)) else {
             return nil
         }
-
+        
```

Comment:
Can we remove bonus space like the? I think there’s a setting in Xcode that will keep this from happening. I haven’t been able to setup a pre-commit linter yet